lighthouse-core/test/computed/metrics/timing-summary-test.js

```diff
@@ -18,6 +18,7 @@ describe('Timing summary', () => {
 
     expect(result.metrics).toMatchInlineSnapshot(`
 Object {
+  "cumulativeLayoutShift": undefined,
```

Comment:
would be nice to have at least one trace with this event in there as a test that isn't sample-json

Reply:
good call. i got this pending in a followup to simplify.